.github/workflows/release-cli.yml

```diff
@@ -86,9 +91,140 @@ jobs:
           name: artifacts-${{ matrix.GOOS }}-${{ matrix.GOARCH }}
           path: bin/*
 
+  package-deb:
+    name: Build deb package
+    needs: build
+    strategy:
+      matrix:
+        include:
+          - GOOS: linux
+            GOARCH: amd64
+            pkgArch: amd64
+          - GOOS: linux
+            GOARCH: arm64
+            pkgArch: arm64
+    runs-on: ubuntu-24.04
+    steps:
+      - name: Get tag name
+        uses: olegtarasov/get-tag@2.1.3
+        id: tagName
+        with:
+          tagRegex: "cli/v(.*)"
+      - name: Download artifacts
+        uses: actions/download-artifact@v4
+        with:
+          path: bin
+          pattern: artifacts-${{ matrix.GOOS }}-${{ matrix.GOARCH }}
+          merge-multiple: true
+
+      - name: create dir structure for .deb package
+        run: |
+          mkdir -p .debpkg/usr/bin
+
+          # when we download from deb, the binary should be named kubetail
+          mv 'bin/kubetail-${{ matrix.GOOS }}-${{ matrix.GOARCH }}' bin/kubetail 
+          cp -p bin/kubetail .debpkg/usr/bin
+
+      - name: create a .deb package
+        uses: jiro4989/build-deb-action@v4
+        with:
+          package: kubetail
+          package_root: .debpkg
+          maintainer: Andres Morey
+          version: ${{ steps.tagName.outputs.tag }}
+          arch: ${{ matrix.pkgArch }}
+          desc: "Real-time logging dashboard for Kubernetes"
+
+      - name: Calculate sha256 checksum
+        run: |
+          FILE='./kubetail_${{ steps.tagName.outputs.tag }}_${{ matrix.pkgArch }}.deb'
+          OUTPUT_FILE=${FILE}.sha256
+          sha256sum "$FILE" | cut -d " " -f 1 >"$OUTPUT_FILE"
+          mkdir -p deb-output
+          mv ./kubetail* ./deb-output/
+
+      - name: Upload .deb artifacts
+        uses: actions/upload-artifact@v4
+        with:
+          name: artifacts-${{ matrix.pkgArch }}-deb
+          path: deb-output/*
+
+  package-rpm:
+    name: Build rpm package
+    needs: build
+    strategy:
+      matrix:
+        include:
+          - GOOS: linux
+            GOARCH: amd64
+            pkgArch: x86_64
+          - GOOS: linux
+            GOARCH: arm64
+            pkgArch: aarch64
+    runs-on: ubuntu-24.04
+    steps:
+      - name: Get tag name
+        uses: olegtarasov/get-tag@2.1.3
+        id: tagName
+        with:
+          tagRegex: "cli/v(.*)"
+      - uses: actions/checkout@v4
+      - name: Download artifacts
+        uses: actions/download-artifact@v4
+        with:
+          path: bin
+          pattern: artifacts-${{ matrix.GOOS }}-${{ matrix.GOARCH }}
+          merge-multiple: true
+
+      - name: Rename the build binary to kubetail
+        run: |
+          mv 'bin/kubetail-${{ matrix.GOOS }}-${{ matrix.GOARCH }}' bin/kubetail
+
+      - name: install the rpm and rpm build
+        run: sudo apt install rpm build-essential
```

Comment:
Some suggestions:
- You should run `update` before installing things from the respository
- I think you need to use `install -y` to automatically approve downloads
- Doesn't the ubuntu runner use "apt-get"?
- I'm used to seeing dependency management come in earlier steps. I would do this before downloading artifacts

```console
sudo apt-get update
sudo apt-get install -y rpm build-essential
```

Reply:
As far as I know "apt-get" is for Ubuntu and "apt" is for Alpine but maybe the Ubuntu runners aliased "apt"